src/isomorphic/classic/types/__tests__/ReactPropTypes-test.js

```diff
@@ -591,11 +591,13 @@ describe('ReactPropTypes', function() {
 
   describe('OneOf Types', function() {
     it('should fail for invalid argument', function() {
-      typeCheckFail(
-        PropTypes.oneOf('red', 'blue'),
-        'red',
-        'Invalid argument supplied to oneOf, expected an instance of array.'
-      );
+      spyOn(console, ['error']);
+
+      PropTypes.oneOf('red', 'blue');
+
+      expect(console.error).toHaveBeenCalled();
+      expect(console.error.calls.argsFor(0)[0])
+        .toContain('Invalid argument supplied to oneOf, expected an instance of array.');
```

Comment:
Let’s additionally test that type check passes (since that’s the behavior we have now).